cmd/dockerd/grpclog.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc/grpclog"
+)
+
+// grpc's default logger is *very* noisy and uses "info" and even "warn" level logging for mostly useless messages.
+// This function configures the grpc logger to step down the severity of all messages.
+//
+// info => trace
+// warn => debug
+// error => warn
+func configureGRPCLog() {
+	l := logrus.NewEntry(logrus.StandardLogger()).WithField("library", "grpc")
```

Comment:
```suggestion
	l := logrus.WithField("library", "grpc")
```

Reply:
/me waits for the flood of github actions cancellation notifications.